### PR TITLE
Add access widener for SoundEvent.of

### LINKED ISSUE #2694: SoundEvent turned private
In 1.19.3-pre2 the SoundEvent class is private and thus you are unable register SoundEvents as described here https://fabricmc.net/wiki/tutorial:sounds
The static constructor methods of SoundEvent also seem to be protected. The SoundEvents class can register them due to being in the same package but as a modder it seems like we need to use AW or a Mixin now.

### PR DESCRIPTION
**Pull request by modmuss50**
Fixes #2694

### CODE REVIEW

**Review by apple502j — APPROVED** [review]

File: fabric-transitive-access-wideners-v1/template.accesswidener [hunk 1]
```diff
@@ -113,4 +113,8 @@ transitive-accessible method net/minecraft/village/VillagerType <init> (Ljava/la
 # Access stack list in SimpleInventory
 transitive-accessible field net/minecraft/inventory/SimpleInventory stacks Lnet/minecraft/util/collection/DefaultedList;
 
+# Creating custom Sound event's
```
Comment (Juuxel) [inline fabric-transitive-access-wideners-v1/template.accesswidener]:
```suggestion
# Creating custom sound events
```

File: fabric-transitive-access-wideners-v1/src/main/resources/fabric-transitive-access-wideners-v1.accesswidener [hunk 2]
```diff
@@ -113,6 +113,10 @@ transitive-accessible method net/minecraft/village/VillagerType <init> (Ljava/la
 # Access stack list in SimpleInventory
 transitive-accessible field net/minecraft/inventory/SimpleInventory stacks Lnet/minecraft/util/collection/DefaultedList;
 
+# Creating custom Sound event's
```
Comment (Shnupbups) [inline fabric-transitive-access-wideners-v1/src/main/resources/fabric-transitive-access-wideners-v1.accesswidener]:
```suggestion
# Creating custom sound events
```

Comment (modmuss50) [reply fabric-transitive-access-wideners-v1/src/main/resources/fabric-transitive-access-wideners-v1.accesswidener]: Good thinking, we should add something to check that there arent any generated file chages 🤔

Comment (Juuxel) [reply fabric-transitive-access-wideners-v1/src/main/resources/fabric-transitive-access-wideners-v1.accesswidener]: Seems simple enough, I could make a PR for a task to check that the real generated file matches the output

**Review by oliviathevampire — APPROVED** [review]

**Review by Shnupbups — APPROVED** [review]

**Pull request merged**